docs/getting_started/setup.rst

```diff
@@ -21,14 +21,22 @@ See the `installing Helm <https://helm.sh/docs/intro/install/>`_ page for other
    pip install astro-provider-ray
 
 
-**3. (Optional) Set up Cloud Authentication**
+**3. (Optional) Set up RayCluster and Airflow**
+
+.. toctree::
+   :maxdepth: 1
+
+   Setup RayCluster and Airflow <kind_ray>
+
+
+**4. (Optional) Set up Cloud Authentication**
 
 .. toctree::
    :maxdepth: 1
 
    Amazon Web Services (AWS) <cloud_platforms_auth>
 
-**3. Setting up the Airflow connection**
+**5. Setting up the Airflow connection**
```

Comment:
We're splitting up the details on setting up the connection in 2 parts. One here & one in the `kind_ray.rst`. Could we isolate this to a single place (probably just here?)

Reply:
This document looks more like a user guide than a contributor's document. I have removed the reference from it and added it to the contributing document.